{"uuid": "e86c9da8-d05e-4408-b968-6ef9d1da6e37", "children": ["d1502916-7e2c-41cb-bbe8-6bfbb4339ccc"], "befores": [{"name": "browser", "status": "passed", "start": 1748793036778, "stop": 1748793038131}], "afters": [{"name": "browser::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"E:\\Dev2\\tests\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"E:\\Dev2\\tests\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"E:\\Dev2\\tests\\conftest.py\", line 26, in browser\n    browser.quit()\n  File \"E:\\Dev2\\tests\\venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 221, in quit\n    self.service.stop()\n  File \"E:\\Dev2\\tests\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 157, in stop\n    self.send_remote_shutdown_command()\n  File \"E:\\Dev2\\tests\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 137, in send_remote_shutdown_command\n    request.urlopen(f\"{self.service_url}/shutdown\")\n  File \"C:\\Python312\\Lib\\urllib\\request.py\", line 212, in urlopen\n    _opener = opener = build_opener()\n                       ^^^^^^^^^^^^^^\n  File \"C:\\Python312\\Lib\\urllib\\request.py\", line 593, in build_opener\n    opener.add_handler(klass())\n                       ^^^^^^^\n  File \"C:\\Python312\\Lib\\urllib\\request.py\", line 1386, in __init__\n    context = http.client._create_https_context(http_version)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python312\\Lib\\http\\client.py\", line 807, in _create_https_context\n    context = ssl._create_default_https_context()\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python312\\Lib\\ssl.py\", line 712, in create_default_context\n    context.load_default_certs(purpose)\n  File \"C:\\Python312\\Lib\\ssl.py\", line 533, in load_default_certs\n    self._load_windows_store_certs(storename, purpose)\n  File \"C:\\Python312\\Lib\\ssl.py\", line 522, in _load_windows_store_certs\n    self.load_verify_locations(cadata=cert)\n"}, "start": 1748793041068, "stop": 1748793041154}], "start": 1748793036778, "stop": 1748793041154}